forte/processors/data_augment/algorithms/eda_processors.py

```diff
@@ -226,18 +230,20 @@ def default_configs(cls):
         Returns:
             A dictionary with the default config for this processor.
             Additional keys for Random Deletion:
-            - alpha: the probability to delete each word
+            - alpha: 0 <= alpha <= 1. The probability to delete each word.
         """
         config = super().default_configs()
         config.update({
             'augment_entry': "ft.onto.base_ontology.Token",
             'other_entry_policy': {
+                'type': '',
```

Comment:
I think we miss the documentation of these configs. For example, I don't know what `type` does here.

Reply:
Because `texar\torch\hyperparams.py`, line 203 requires that "'kwargs' must accompany with 'type'". To add to the pipeline we need the `type` parameter.

Reply:
It would be helpful to add a comment here to state that? Basically this is a hack to use Texar hyperparams